docs/cse/sensors/network-sensor-deployment-guide.md

```diff
@@ -320,21 +295,13 @@ Description. The directory to which the sensor writes files extracted from netwo
 
 ### filter
 
-**Description.** This option tells bro not to capture network traffic between the sensor and the Sumo Logic (or the CSE server, in legacy environments). The sensor sends status reports and bro output files to end points on the destination, which we don’t want bro to capture. 
+**Description.** This option tells bro not to capture network traffic between the sensor and the Sumo Logic. The sensor sends status reports and bro output files to end points on the destination, which we don’t want bro to capture. 
```

Comment:
@josh-williams @jpipkin1 double checking that `bro` is the correct terminology here!

Reply:
@josh-williams, you should change instances of "bro" to "Bro" when you are talking about the application. @kimsauce, I looked it up, and found that Bro is the older name of an open-source network traffic analyzer, which is now known as Zeek.

Reply:
Fixed.